packages/react/src/profiler.tsx

```diff
@@ -1,7 +1,7 @@
-import { getCurrentHub, Hub } from '@sentry/browser';
-import { Integration, IntegrationClass, Span, Transaction } from '@sentry/types';
-import { timestampWithMs } from '@sentry/utils';
-import hoistNonReactStatics from 'hoist-non-react-statics';
+import { getCurrentHub } from '@sentry/browser';
+import { Integration, IntegrationClass, Span } from '@sentry/types';
+import { logger, timestampWithMs } from '@sentry/utils';
+import * as hoistNonReactStatic from 'hoist-non-react-statics';
```

Comment:
These changes to React package seem to be unrelated to adding Angular (?). Can we move it off to a separate PR?

Reply:
Yes please make sure this is changed, we want to still support `@sentry/tracing` with the Profiler.